### DISCUSSION TITLE
v1.8.0 Release Notes

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#1677] [Announcements]
# News

## Beta Features

![Screenshot 2023-06-19 at 2 00 00 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/e3fd09d3-7b86-44dc-9e4b-2726b7975f3f)

Starting today, new Skeleton features will include a small `Beta` tag in the documentation navigation. This allows us to launch new features quickly, but still make breaking changes based on your feedback. If you are leery of breaking changes, please avoid using beta features in production. However, in most cases you can expect the beta tag to be removed in the following release cycle.

# Release Notes

- [View Release](https://github.com/skeletonlabs/skeleton/releases/tag/%40skeletonlabs%2Fskeleton%401.8.0)
- [View Changelog](https://github.com/skeletonlabs/skeleton/blob/master/packages/skeleton/CHANGELOG.md)

After several maintenance focused updates, we return with a new set of feature updates. This includes a brand new and interactive ratings component, new layouts for handling images and more, introducing an anchor-specific variation for tabs, and several new features for Toast notifications. If you wish to learn more about each new feature, continue reading below. Tap the Changelog link above if you wish to see the full list of changes.

Summary:

- ⭐ Ratings component added
- 🖼️ Image Layout blocks added
- 🔗  TabAnchor component added
- 🍞 Expanded Toast features
- 🐞 Plus general improvements and bugfixes 

## Ratings Component (Beta) - https://github.com/skeletonlabs/skeleton/pull/1654

![Screenshot 2023-06-19 at 1 50 47 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/9d7bc2b0-91ba-408d-b903-7a394c93ddd7)

This component allows you to create a visual representation of numeric values - perfect for ratings for movies and more. This supports SVG, emoji, images, and text per each of the whole, half, and empty states. Plus see the documentation to learn how to make this interactive on click.

https://skeleton.dev/components/ratings

## Image Layouts - https://github.com/skeletonlabs/skeleton/pull/1617

![Screenshot 2023-06-19 at 1 56 24 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/3975f1ba-b767-4716-8a8d-ac2ea24064fd)

This new Blocks section pairs HTML markup with standard Tailwind utility classes to provide grid or masonry layouts for displaying images. The styling for this feature comes directly from our friends over at [Flowbite](https://flowbite.com/). If you're Interested in learning more about how to pair Skeleton and Flowbite, [see our dedicated guide](https://www.skeleton.dev/blog/skeleton-plus-flowbite).

https://skeleton.dev/elements/image-layouts

## TabAnchor Component

![Screenshot 2023-06-19 at 2 02 20 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/05440004-25db-4012-a4f1-fb7b5b4372c2)

Following the same pattern as the [AppRailAnchor component in Skeleton v.1.6](https://github.com/skeletonlabs/skeleton/discussions/1540), we've now introduced a TabAnchor component. This provides dedicated support for anchor links that are displayed as Tabs. Make sure to read the documentation to understand the ideal use case between standard Tabs and TabAnchors, as these should not be used interchangeably.

https://skeleton.dev/components/tabs

## New Toast Features

https://github.com/skeletonlabs/skeleton/assets/1509726/20f38762-29f7-4b28-9216-d8d300a2f1a1

The Toast notifications feature has been expanded with a number of new features, including:

- Now supports hiding the dismiss button when using the `hideDismiss` option
- Can remain visible when hovering with a mouse cursor using the `hoverable` option
- Has the ability to be closed programmatically using each Toast's unique reference ID

See the revamped Toast documentation to learn more about each of these features:
https://skeleton.dev/utilities/toasts

**Comment by derekhearst** [comment]
Looks amazing! Thank you for your continued support and development of this amazing library : )

**Reply by endigo9740** [reply]
Thank you for your kind words and use of the library! :) A lot of updates from today came from @Mahmoud-zino so he deserves the shoutout tbh!

**Comment by geoffkendall** [comment]
Another cracking update. Thanks @endigo9740 and @Mahmoud-zino !
